cmd/ftl/cmd_call.go

```diff
@@ -8,6 +8,7 @@ import (
 
 	"connectrpc.com/connect"
 	"github.com/jpillora/backoff"
+	lev "github.com/texttheater/golang-levenshtein/levenshtein"
```

Comment:
Rather than introduce this dependency, which has a _very_ odd `[]rune` API, what I've done in the past is copy [this code](https://en.wikibooks.org/wiki/Algorithm_Implementation/Strings/Levenshtein_distance#Go) with attribution. Let's do the same here. Can get rid of the `min()` function in that code as it's built in to Go now.

Reply:
yea, agreed. was lazy. wanted to demo it haha. will update

Reply:
btw there's a reason for the rune - doing the distance for unicode. of course we don't need unicode here, but afaik the distance can be thrown off by unicode characters.

Reply:
The linked function uses runes internally, but accepts strings.